internal/handler/msg_serverstatus.go

```diff
@@ -69,6 +70,41 @@ func (h *Handler) msgServerStatus(connCtx context.Context, req *middleware.Reque
 		}
 	}
 
+	// Add CommandFeatureCounterStats as Prometheus metrics
+	featureCounterStatsDoc := wirebson.MakeDocument(0)
+
+	// Try to get feature counter stats, but don't fail if the function doesn't exist
+	err = h.Pool.WithConn(func(conn *pgx.Conn) error {
+		// Query for all feature counter stats
+		rows, queryErr := conn.Query(connCtx,
+			`SELECT feature_name, usage_count FROM documentdb_api_internal.command_feature_counter_stats($1)`,
+			false,
```

Comment:
We have duplicated code, could we have a common code possibly in `internal/documentdb` and use that?